poznaj/stories/tests/test_views.py

```diff
@@ -76,3 +79,45 @@ def test_update_story(self):
         self.assertEqual(database_story.title, 'new_title')
         self.assertEqual(database_story.description, 'new_description')
         self.assertEqual(str(database_story.duration), '1:00:00')
+
+    def test_filter_story(self):
```

Comment:
@krzysztofzuraw  
mamy robić hackaton z miastem :+1: 
we wtorek idę gadać o tym do siedziby wroclaw.pl
pokaże im androidową apkę :)